src/resources/CrawlingModule/CrawlingModule.ts

```diff
@@ -0,0 +1,31 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {ComponentVersion, CrawlingModuleEntity, MaestroVersionOptions, UpdateStatus} from './CrawlingModuleInterfaces';
+
+export default class CrawlingModule extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/crawlingmodule`;
+
+    getCrawlingModules() {
```

Comment:
[optional] Is it better to name it directly `list()`? Because we get an array of `CrawlingModuleEntity` and we use `baseUrl` here, also we can call it simply like `Platform.crawlingModule.list()`.

Reply:
good point. Thanks!